### PR TITLE
Improving wording for voiceStatusCallback in Numbers API

### PR DESCRIPTION
**Pull request by tommorris**
Fixing incorrect title as well

### CODE REVIEW

File: _api/developer/numbers.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Developer - Numbers API Reference
+title: API reference
 description: Reference guide for the Account API.
 api: Developer API
 ---
 
-# Developer - Numbers API Reference
+# API reference
```
Comment (adambutler) [inline _api/developer/numbers.md]: Why the change here? These were changed a little while ago to include the context so that the page is clearer when searching.

Comment (tommorris) [reply _api/developer/numbers.md]: I thought that was the approach we were going with. I'll flip it back.
